ivadomed/models.py

```diff
@@ -325,14 +324,13 @@ def __init__(self, in_channel=1, depth=3, drop_rate=0.4, bn_momentum=0.1, n_meta
         self.down_path.append(max_pool(2))
 
         for i in range(depth - 1):
-            self.down_path.append(DownConv(n_filters, n_filters * 2, drop_rate, bn_momentum, is_2d))
-            self.down_path.append(FiLMlayer(n_metadata, n_filters * 2) if film_layers and film_layers[i + 1] else None)
+            self.down_path.append(DownConv(n_filters*2**i, n_filters*2**(i+1), drop_rate, bn_momentum, is_2d))
+            self.down_path.append(FiLMlayer(n_metadata, n_filters * 2**(i+1)) if film_layers and film_layers[i + 1] else None)
             self.down_path.append(max_pool(2))
-            n_filters = n_filters * 2
 
         # Bottom
-        self.conv_bottom = DownConv(n_filters, n_filters, drop_rate, bn_momentum, is_2d)
```

Comment:
In the U-net paper, in the bottom layer, the number of filters doubled but I think in this line the number of filters (output channels) remained the same as the input one.